src/app/events/[event]/participants/data-participants.ts

```diff
@@ -53,3 +57,33 @@ export async function checkParticipant({
     },
   })
 }
+
+interface RegisterParticipantInput {
+  event: string
+  data: RegisterParticipant
+}
+
+export async function registerParticipant({
+  event,
+  data,
+}: RegisterParticipantInput) {
+  const result = await fetch(
+    `/api/events/${event}/participants/register-participants`,
```

Comment:
Since you're registering one participant at a time, the route should be named as `register-participant`, singular.